articles/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Enabling HTTPS Engagement Tracking on SparkPost"
+description: "SparkPost supports HTTPS engagement tracking for customers via self-service for all SparkPost customers. To enable SSL engagement tracking for a domain, additional configuration for SSL keys is required."
+---
+
+## Overview
+
+SparkPost supports HTTPS engagement tracking for customers via self-service for all SparkPost customers. To enable SSL engagement tracking for a domain, additional configuration for SSL keys is required.
+
+## Configuring SSL Certificates
+
+In order for HTTPS engagement tracking to be enabled on SparkPost, our service needs to present a valid certificate that will be trusted by the email recipient’s browser.  SparkPost does not manage certificates for customer engagement tracking domains, as we are not the record owner for our customers’ domains. 
+
+As a workaround, you may use a Content Delivery Network (CDN) service, such as CloudFlare or Fastly to manage certificates and keys for any custom engagement tracking domains you configure.  These services forward traffic onwards to SparkPost so that HTTPS tracking can be performed. 
+
+## Step by Step Guide with CloudFlare
+
+The following is a sample guide for use with CloudFlare **only**; please note, the steps to configure your chosen CDN will likely differ from CloudFlare in workflow. Please refer to your CDN's documentation and contact their respective support departments if you have any questions.
+
+1.	Create CloudFlare account
+2.	Go to “DNS” tab on the CloudFlare UI
+
+![](media/enabling-https-engagement-tracking-on-sparkpost/cloudflare%20UI.png)
+
+3.	Add domain and then add the following Cloudflare NS records (**please note**, for other providers, the NS records to be used will differ):
+  
+  	NS	aron.ns.cloudflare.com
+	
+  	NS	peyton.ns.cloudflare.com (these values can be found under the DNS tab on the Cloudflare UI) 
+
+	**Example:**
+
+	Using the domain "isaackim.info", below is a command line DIG command to confirm that the NS records have been updated to reflect the required changes:
+
```

Comment:
Needs an indent to keep it under the list item above. Same goes for the content under items 4, 5 and 6.

If you use the [Atom](https://atom.io/) editor, it'll give you a live Markdown preview which makes it easier to catch this type of formatting issue.